### DISCUSSION TITLE
How to recover after forgetting the root password?

### DISCUSSION THREAD

**Discussion by nsd-chenwenfei** [ccfos/nightingale#1169] [Q&A] [answered]
Nightingale V5版本部署后忘记root密码后如何解决？请教教我，谢谢！

**Comment by UlricQin** [comment, marked as answer]
数据库有个 users 表，如果除了root账号，还有其他账号，直接 update 数据库，把 root 密码设置为其他账号的密码，用其他账号的密码登录 root 账号，再修改 root 密码即可。如果数据库里只有 root 账号，直接修改 root 密码，设置为 root.2020 明文，重启webapi，webapi启动的时候发现 root 密码不是加密的，会自动加密，然后就可以用 root.2020 登录了

**Comment by nsd-chenwenfei** [comment]
已解决
